spec/scoreCalculatorSpec.js

```diff
@@ -0,0 +1,183 @@
+describe('ScoreCalculator', function(){
+
+  var scoreCalculator;
+
+  beforeEach(function(){
```

Comment:
If you're only using each game in one test it'd be clearer to assign it right next to where it's being used (and they don't all need to be assigned before every test).  Should you find that you're using a given game in more than one test and you want to DRY up your test suite, it might be clearer to use a descriptive variable name – these are quite complex data sets though so that could be hard :)